03-base-r.Rmd

```diff
@@ -2,9 +2,6 @@
 
 ```{r base-r-setup, include = FALSE}
 knitr::opts_chunk$set(fig.path = "figures/")
-library(tidyverse)
-library(lubridate)
-library(tidymodels)
```

Comment:
I've been avoiding having the `library(tidymodels)` output in the book (since they take up space). Can you add these back in?

Reply:
Sure thing. Do you know why lubridate was loaded? I can't find any uses of it.

Reply:
Hmm. I think that was used by the sections that I removed.